dotcom-rendering/index.d.ts

```diff
@@ -565,7 +565,7 @@ declare namespace JSX {
 	interface IntrinsicElements {
 		'gu-island': {
 			name: string;
-			deferUntil?: 'idle' | 'visible' | 'interaction';
+			deferUntil?: 'idle' | 'visible' | 'interaction' | 'feature';
```

Comment:
This may not be apltly named…

Reply:
Perhaps: `defer-until="feature-flag"`?

Reply:
just seen that you went for `hash` -- that sounds right to me; good to be explicit!